java/java.hints/test/unit/src/org/netbeans/modules/java/hints/jdk/ConvertToPatternInstanceOfTest.java

```diff
@@ -140,15 +148,4 @@ public void testNoSoSimpleNameClash() throws Exception {
                               "}\n");
     }
 
-    @Override
```

Comment:
We are giving up the ability to run tests without nb-javac on JDK<14. Possibly not that big issue as we are not running the tests in that combination anyway.